keyboards/handwired/ergodox_mass/ergodox_mass.h

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2018 Andrew Mass
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#define LAYOUT(                      \
```

Comment:
You could (and probably should) put the halves side by side.

If you look at the Ergodox EZ's `LAYOUT_ergodox_pretty`, you'll see what I mean. 

https://github.com/qmk/qmk_firmware/blob/master/keyboards/ergodox_ez/ergodox_ez.h#L197-L224

Reply:
I had the layout defined that way originally. The problem I ran into was that the lines were much too long (>160chars) and were wrapping around. I think this way is similar to what the default ergodox keymaps use now.

Reply:
Well, that should be okay for the layout macros. 

And honestly, IMO, it looks better, especially to anyone else that may use the board.  But it's not a problem, either way.